features/docs/formatters/junit_formatter.feature

```diff
@@ -187,6 +228,38 @@ Feature: JUnit output formatter
 
       """
 
+  Scenario: pending and undefined steps add fileattribute
+    When I run `cucumber --format junit,fileattribute=true --out tmp/ features/pending.feature`
+    Then it should pass with:
+      """
+
+      """
+    And the junit output file "tmp/TEST-features-pending.xml" should contain:
+      """
+      <?xml version="1.0" encoding="UTF-8"?>
+      <testsuite failures="0" errors="0" skipped="2" tests="2" time="0.05" name="Pending step">
```

Comment:
how brittle are these timestamps? I think Vincent had something that regex'd them. Could we use something like that here?

Reply:
These timestamps are indeed regex'd to `0.05`

https://github.com/cucumber/cucumber-ruby/blob/master/features/lib/support/junit.rb

And applied to any of those steps:
https://github.com/cucumber/cucumber-ruby/blob/master/features/lib/step_definitions/junit_steps.rb

Because of the above, it is my understanding that this would be stable as well.

Reply:
Perfect, yes sorry I wasn't aware of that bit of the codebase. Perfectly fine :+1: